gef-ui/server/index.js

```diff
@@ -105,13 +105,13 @@ app.use('/', routes);
 app.use('/assets', express.static(path.join(__dirname, '..', 'public')));
 
 // eslint-disable-next-line no-unused-vars
```

Comment:
do we need this here?

Reply:
All of these will be removed as part of separate PR.